config/features.yml

```diff
@@ -1449,10 +1449,6 @@ features:
     actor_type: user
     description: >
       When enabled, 21P-527EZ will display additional explanations for the income and assets requirement.
-  pension_income_and_assets_overflow_pdf_redesign:
-    actor_type: user
-    description: >
-      When enabled, 21P-0969 will use the new extras redesign when generating the overflow pdf.
   pension_medical_evidence_clarification:
```

Comment:
Once this is deployed, you'll need to follow step 4 of [this section of this guide](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Removingold/unusedfeaturetoggles) to completely remove this toggle. You'll need to do it in the terminal of each ArgoCD environment. If you or no one on your team has access to those terminals, let me know and I can do it for you. Remember, don't delete them in Argo until the code has been deployed.

Reply:
Sounds good!